.gitignore

```diff
@@ -1,6 +1,4 @@
-# IntelliJ IDE
-/package-url-java.iml
```

Comment:
I need `/package-url-java.iml`
```
$ git status 
On branch RemoveGuava
Your branch is up to date with 'origin/RemoveGuava'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        package-url-java.iml

nothing added to commit but untracked files present (use "git add" to track)
```

Maybe you have it ignored globally, but that is not so in my case.